### PR TITLE
Unisex ruling positions

### PR DESCRIPTION
**Pull request by Gyrandola**
## About The Pull Request

Allows female characters to roll the "Duke" role.
Allows male characters to roll the "Duchess" role, now renamed to "Duke Consort / Duchess Consort".

Prevents same-sex scenarios by only matching consorts of the opposite gender of the ruler.

### CODE REVIEW

**Comment by wabbajacked** [comment]
There will be no equality in MY ratwood.

File: code/modules/jobs/job_types/roguetown/nobility/consort.dm [hunk 1]
```diff
@@ -0,0 +1,135 @@
+/datum/job/roguetown/consort
+	title = "Duke Consort"
+	f_title = "Duchess Consort"
+	flag = LADY
```
Comment (Heheniy) [inline code/modules/jobs/job_types/roguetown/nobility/consort.dm:4]: not really lady anymore

**Comment by Heheniy** [comment]
female ruler having old consort traits and male consort having duke traits is kinda... meh...

**Comment by Frienderanian** [comment]
But why tho?

**Comment by Gyrandola** [comment]
> female ruler having old consort traits and male consort having duke traits is kinda... meh...

This is not a balance PR

**Review by EvenInDeathIStillServe — APPROVED** [review]

**Pull request merged**